modules/develop/pages/transactions.adoc

```diff
@@ -260,19 +256,39 @@ while (true) {
     }
 }
 ----
+====
 
-Different transactions require different approaches to handling failures within the application. Consider the approaches to failed or timed-out transactions in the provided use cases.
-
-* Publishing of multiple messages: The request came from outside the system, and it is the application's responsibility to discover the true status of a timed-out transaction. (This example doesn't use consumer groups to distribute partitions between consumers.)
-* Exactly-once streaming (consume-transform-loop): This is a closed system. Upon re-initialization of the consumer and producer, the system automatically discovers the moment it was interrupted and continues from that place. Additionally, this automatically scales by the number of partitions. Run another instance of the application, and it starts processing its share of partitions in the source topic.
+==== Exactly-once processing configuration requirements
 
-== Enabling exactly-once processing
-
-The default configuration of Redpanda supports exactly-once processing. Preserving this capability requires maintaining the following settings:
+Redpanda’s default configuration supports exactly-once processing. To preserve this capability, ensure the following settings are maintained:
 
 * `enable_idempotence = true`
 * `enable_transactions = true`
-* `transaction_coordinator_delete_retention_ms >= transactional_id_expiration_ms`
+* `transaction_coordinator_delete_retention_ms` is greater than or equal to `transactional_id_expiration_ms`
```

Comment:
This is specified in the [transaction_coordinator_delete_retention_ms](https://deploy-preview-1257--redpanda-docs-preview.netlify.app/current/reference/properties/cluster-properties/#transaction_coordinator_delete_retention_ms), is it correct to call it out here? Is "To ensure transaction state is retained as long as the longest-running transaction" commonly recommended / best practice?